src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiConfigurationOptions.cs

```diff
@@ -129,11 +132,20 @@ protected static bool IsHttpForced()
         /// <returns>Returns <c>True</c>, if HTTPS is forced; otherwise returns <c>False</c>.</returns>
         protected static bool IsHttpsForced()
         {
-            var development = bool.TryParse(Environment.GetEnvironmentVariable(ForceHttpsKey), out var result) ? result : false;;
+            var development = bool.TryParse(Environment.GetEnvironmentVariable(ForceHttpsKey), out var result) ? result : false;
 
             return development;
         }
 
+        /// <summary>
+        /// Adds an instance of the given <see cref="IDocumentFilter"/> type.
+        /// </summary>
+        /// <typeparam name="T">Type of <see cref="IDocumentFilter"/>.</typeparam>
+        protected void AddDocumentFilter<T>() where T : IDocumentFilter, new()
```

Comment:
is this needed? it feels like only a convenience method, which could be moved out as an extension method
it is a bit odd for this to have lower visibility than `DocumentFilters`.

Reply:
You're right, this is a bit odd. I've removed the method, using `this.DocumentFilters.Add(new Something())` is just fine.